modules/react-native-status/ios/RCTStatus/RCTStatus.m

```diff
@@ -102,6 +102,18 @@ -(void)setBridge:(RCTBridge *)newBridge
     });
 }
 
+//////////////////////////////////////////////////////////////////// executeJS
+RCT_EXPORT_METHOD(executeJS:(NSString *)chatId
```

Comment:
Did you try this with existing version of `status-go` develop? I would expect this to fail if executeJS isn't exposed in status-go yet

Reply:
Oh so this is in status-go develop already https://github.com/status-im/status-go/blob/develop/lib/library.go#L343 :)